GitUI/BuildServerIntegration/BuildServerWatcher.cs

```diff
@@ -44,55 +45,59 @@ public BuildServerWatcher(RevisionGrid revisionGrid, DvcsGraph revisions)
 
         public void LaunchBuildServerInfoFetchOperation()
         {
+            Debug.Assert(ThreadHelper.JoinableTaskContext.IsOnMainThread);
+
             CancelBuildStatusFetchOperation();
 
             DisposeBuildServerAdapter();
 
             // Extract the project name from the last part of the directory path. It is assumed that it matches the project name in the CI build server.
-            GetBuildServerAdapter().ContinueWith((Task<IBuildServerAdapter> task) =>
-            {
-                if (_revisions.IsDisposed)
+            ThreadHelper.JoinableTaskFactory.RunAsync(
+                async () =>
                 {
-                    return;
-                }
+                    _buildServerAdapter = await GetBuildServerAdapterAsync().ConfigureAwait(false);
```

Comment:
I see here a sequence of ConfigureAwait(false) and SwitchToMainThreadAsync. On the other hand there is no ConfigureAwait(false) in the FormBrowse.cs at the 1127 line. Is it a mistake or is it done on purpose?

Reply:
I will remove the `ConfigureAwait(false)` here